annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/ClassDescription.java

```diff
@@ -0,0 +1,29 @@
+package com.exactpro.epfast.annotation.processing;
+
+public class ClassDescription {
+    private String packageName;
+
+    private String className;
+
+    public ClassDescription(String packageName, String className) {
+        this.packageName = packageName;
+        this.className = className;
+    }
+
+    public String getPackageName() {
+        return packageName;
+    }
+
+    public String getClassName() {
+        return className;
+    }
+
```

Comment:
this empty line is excessive. remove it and add checkstyle rule to catch such things. see https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.html